plexupdate.sh

```diff
@@ -136,7 +138,9 @@ usage() {
 	echo "    --pass <plex.tv password> Plex.TV password"
 	echo "    --server <Plex server address> Address of Plex Server"
 	echo "    --port <Plex server port> Port for Plex Server. Used with --server"
-	echo
+	echo ""
+	echo "    --notify-success Makes plexupdate return 10 on download/update success (instead of 0)"
```

Comment:
The language here doesn't match the other descriptions. I'd suggest something like "Return exit code 10 if an update is available/installed"